### DISCUSSION TITLE
how to use <c-w> to delete word in commit message prompt

### DISCUSSION THREAD

**Discussion by antosha417** [jesseduffield/lazygit#1987] [Q&A] [answered]
Hi! 
I'm used to delete a word in terminal or insert mode in vim with <c-w> shortcut.
But it doesn't work for me in commit message prompt. 

Do you know how can  I make it work?

**Comment by Ryooooooga** [comment, marked as answer]
https://github.com/jesseduffield/lazygit/issues/1783
